src/Cards/SimpleCard.php

```diff
@@ -15,7 +15,7 @@ public function getMessage()
         return [
             "@context" => "http://schema.org/extensions",
             "@type" => "MessageCard",
-            "themeColor" => "0072C6",
+            "themeColor" => isset($this->data['themeColor']) ? $this->data['themeColor'] : "0072C6",
```

Comment:
We should check if `themeColor` is not an empty string, but than we can merged it

```suggestion
            "themeColor" => !empty($this->data['themeColor']) ? $this->data['themeColor'] : "0072C6",
```